ui/objects/semantic-ui/js/objects/gallery/Carousel.jsx

```diff
@@ -121,56 +128,49 @@ export const ImgCarousel = ({ imgs }) => {
         ))}
       </Slider>
 
-      <div>
-        <Modal open={modalOpen} onClose={() => setModalOpen(false)} className="custom-modal">
-          <Modal.Content image className="modal-content">
-            <Button icon="chevron left" onClick={handlePrevImage} className="carousel-button left" />
-            <div className="vert-div">
-              <Image
-                src={imagesCollection?.[selectedImageIndex]?.links?.content}
-                className="modal-image"
-              />
-              <p>{fileName(imagesCollection?.[selectedImageIndex])}</p>
-            </div>
-            <Button icon="chevron right" onClick={handleNextImage} className="carousel-button right" />
-            <Button icon="close" onClick={() => setModalOpen(false)} className="close-button" />
-          </Modal.Content>
-        </Modal>
-      </div>
+      <Modal open={modalOpen} onClose={() => setModalOpen(false)}>
+        <Modal.Content image>
+          <Button icon="chevron left black" onClick={handlePrevImage} />
+          <Grid columns={1}>
+            <Image
+              src={imagesCollection?.[selectedImageIndex]?.links?.content}
+            />
+            <Header as="a3">
+              {fileName(imagesCollection?.[selectedImageIndex])}
+            </Header>
+          </Grid>
+          <Button icon="chevron right black" onClick={handleNextImage} />
+        </Modal.Content>
+        <Button
+          icon="close"
+          onClick={() => setModalOpen(false)}
+          className="close transparent"
+        />
+      </Modal>
     </>
   );
 };
 
-
 export const FilesSection = ({ files }) => {
```

Comment:
This should definitely be a separate component in a separate file (what it does is very different from Carousel)